app/controllers/spree/taxons_controller_decorator.rb

```diff
@@ -1,4 +1,5 @@
 Spree::TaxonsController.class_eval do
+
   def show
     if stale? [@taxon, @products, @taxonomies, simple_current_order]
       if params.include?('per_page')
```

Comment:
I thibk we dnt need this per params code here. As retrieve_products function is already handling the pagination. Please confirm